RubeesEat/Pages/UserHomepage.cshtml

```diff
@@ -0,0 +1,21 @@
+@page
+@using System.Globalization
+@inject IBillRepository billRepository
+@inject IPersonRepository personRepository
+
+Hello, @personRepository.GetCurrentUser().FirstName!
+
+
+<ul id="balanceChanges" style="list-style: none">
+    @foreach (var item in billRepository.GetRecentBalanceChanges(personRepository.GetCurrentUser().Id, 10))
+    {
+        @*@item.Amount < 0*@
+        <li>
+
+            <div class="amount" style="display: inline; color: @(item.Amount < 0 ? "red" : "green");">@item.Amount€</div>
+
+            <div class="date" style="display: inline;">@item.Date.ToString("D", CultureInfo.CreateSpecificCulture("en-US"))</div>
```

Comment:
Specifying a culture here might not be what we want. In the future we could allow users to change their preferred language. In such scenarios, we want the date to be formatted in the users culture. When you don't specify a culture, it will use the current culture of the PC to format the string.

Reply:
We have an issue during testing. Some of us get English Culture automatically, some German. So when we run the tests, we get different results.

Reply:
Okay then we leave it as is an will handle this once we add localization :)